src/layouts/Dashboard/components/ReviewRequestModal/RequestOverview/RequestOverview.tsx

```diff
@@ -192,63 +192,131 @@ export const RequestOverview = ({
+        <Menu>
+          <MenuButton
+            as={IconButton}
+            variant="clear"
             icon={<BiFilterAlt />}
-            variant="link"
+            fontSize="1.25rem"
+            aria-label="sort by file type"
           />
-        </Th>
+          <MenuList>
+            <MenuItem
+              icon={
+                // NOTE: Using an Icon component to hook into design system results in a
+                // slightly off center icon, which is why using the base component itself
+                // from react-icons + using the theme is preferred.
+                <BiFileBlank fontSize="1rem" fill={theme.colors.icon.alt} />
+              }
+              iconSpacing="0.5rem"
+              onClick={() => {
+                column.setFilterValue(["page", "image", "file"])
+              }}
+            >
+              <Text textStyle="subhead-2" textColor="text.body">
+                Pages
+              </Text>
+            </MenuItem>
+            <MenuItem
+              iconSpacing="0.5rem"
+              icon={<BiCog fontSize="1rem" fill={theme.colors.icon.alt} />}
+              onClick={() => {
+                column.setFilterValue(["setting"])
+              }}
+            >
+              <Text textStyle="subhead-2" textColor="text.body">
+                Settings
+              </Text>
+            </MenuItem>
+            <MenuItem
+              icon={<BiCompass fontSize="1rem" fill={theme.colors.icon.alt} />}
+              iconSpacing="0.5rem"
+              onClick={() => {
+                column.setFilterValue(["nav"])
+              }}
+            >
+              <Text textStyle="subhead-2" textColor="text.body">
+                Navigation
+              </Text>
+            </MenuItem>
+          </MenuList>
+        </Menu>
       ),
       cell: (props) => getIcon(props.row.original.type),
+      filterFn: "arrIncludesSome",
     }),
     columnHelper.accessor((row) => `${row.name} ${row.path}`, {
       id: "itemName",
+      sortingFn: "textCaseSensitive",
       cell: ({ row }) => (
         <ItemName name={row.original.name} path={row.original.path} />
       ),
-      header: () => (
-        <Th textTransform="capitalize">
-          <Text textStyle="subhead-2" textColor="text.label">
-            Item name
-          </Text>
-        </Th>
+      // NOTE: This is case insensitive and will search through both name + path
+      // This is due to the accessor returning a composite string.
+      // Do note that the path will be joined using a , so for composite search paths
+      // like: /some/folder, the correct search term would be some,folder.
+      filterFn: "includesString",
+      header: ({ column }) => (
+        <Button
+          ml="-0.25rem"
+          variant="link"
+          _hover={{
+            textDecoration: "none",
+          }}
+          onClick={column.getToggleSortingHandler()}
+        >
+          <HStack w="28rem" spacing="0.5rem">
```

Comment:
yes, we should. however, this was done because using a percentage width would cause the table to freely resize between content. this is because our rows are virtualised so the table doesn't know, at render time, what the max width of the _global_ state is - it only knows the max width of the _local_ state (displayed rows). 

This is most evident when we have a short title and a long title that don't appear together. when the long title appears, the header will look fine. however, when the short title appears, the table will resize so that the cell will take width of item name, which looked weirder to me than just having a fixed width throughout. this was acceptable to me due to horizontal scrolling existing on table + we don't guarantee support for smaller screen sizes (kekw) 

will defer to @NatMaeTan on this tho

Reply:
What are you referring to when you say 'Modal'? Is this on the tables @seaerchin